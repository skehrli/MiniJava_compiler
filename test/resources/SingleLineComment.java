// Here is a single line comment that should not be included.
class Comment {
    // Here's another one.
    public static void main(String[] args) { // Here's another one.
        int x = 1; // Here's another one.
        // Here are some tokens that shouldn't be included: length main public static
    }
}
// Here's another one.
