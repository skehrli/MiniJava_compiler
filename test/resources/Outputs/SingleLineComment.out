instead expected token classes are [WHILE, LBRACE, IDENTIFIER]
